package.json

```diff
@@ -1,49 +1,129 @@
-    "extends": "marine/node"
-  }
+	"name": "@discordjs/collection",
+	"version": "0.1.6",
+	"description": "Utility data structure used in Discord.js",
+	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"lint": "eslint test src --ext .ts",
+		"lint:fix": "eslint test src --ext .ts --fix",
+		"prebuild": "npm run lint",
+		"build": "rimraf dist/ && tsc",
+		"pretest": "npm run build",
+		"test": "jest --verbose",
+		"docs": "docgen --jsdoc jsdoc.json --source src/*.ts src/**/*.ts --custom docs/index.yml --output docs/docs.json",
+		"docs:test": "docgen --jsdoc jsdoc.json --source src/*.ts src/**/*.ts --custom docs/index.yml"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/discordjs/collection.git"
+	},
+	"keywords": [
+		"map",
+		"collection",
+		"utility"
+	],
+	"files": [
+		"!**/*.ts",
+		"**/*.d.ts",
+		"!package-lock.json"
+	],
+	"author": "Amish Shah <amishshah.2k@gmail.com>",
+	"license": "Apache-2.0",
+	"bugs": {
+		"url": "https://github.com/discordjs/collection/issues"
+	},
+	"homepage": "https://github.com/discordjs/collection#readme",
+	"engines": {
+		"node": ">=14.0.0"
+	},
+	"devDependencies": {
+		"@babel/cli": "^7.12.10",
+		"@babel/core": "^7.12.10",
+		"@babel/preset-env": "^7.12.11",
+		"@babel/preset-typescript": "^7.12.7",
+		"@commitlint/cli": "^11.0.0",
+		"@commitlint/config-angular": "^11.0.0",
+		"@types/node": "^14.14.21",
+		"@typescript-eslint/eslint-plugin": "^4.13.0",
+		"@typescript-eslint/parser": "^4.13.0",
+		"discord.js-docgen": "discordjs/docgen#ts-patch",
+		"eslint": "^7.17.0",
+		"eslint-config-marine": "^8.1.0",
+		"eslint-config-prettier": "^7.1.0",
+		"eslint-plugin-prettier": "^3.3.1",
+		"husky": "^4.3.7",
+		"jsdoc-babel": "^0.5.0",
+		"lint-staged": "^10.5.3",
+		"prettier": "^2.2.1",
+		"rimraf": "^3.0.2",
+		"ts-jest": "^26.4.4",
+		"typescript": "^4.1.3"
+	},
+	"eslintConfig": {
+		"extends": "marine/prettier/node",
+		"parserOptions": {
+			"project": "./tsconfig.eslint.json"
+		},
+		"rules": {
+			"@typescript-eslint/naming-convention": 0
+		}
+	},
+	"prettier": {
+		"printWidth": 120,
+		"useTabs": true,
+		"singleQuote": true,
+		"quoteProps": "as-needed",
+		"trailingComma": "all",
+		"endOfLine": "lf"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged",
+			"commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+		}
+	},
+	"lint-staged": {
+		"*.js": [
+			"prettier --write"
+		],
+		"*.ts": [
+			"eslint --fix",
+			"prettier --write"
+		],
+		"*.{json,yml,yaml}": [
+			"prettier --write"
+		]
+	},
+	"commitlint": {
+		"extends": [
+			"@commitlint/config-angular"
+		],
+		"rules": {
+			"type-enum": [
+				2,
+				"always",
+				[
+					"chore",
+					"build",
+					"ci",
+					"docs",
+					"feat",
+					"fix",
+					"perf",
+					"refactor",
+					"revert",
+					"style",
+					"test",
+					"types",
+					"wip",
+					"src"
+				]
+			]
+		}
+	},
+	"dependencies": {
+		"@types/jest": "^26.0.20",
+		"jest": "^26.6.3"
+	}
```

Comment:
These are not dependencies at all, they're dev dependencies

Reply:
fixed in https://github.com/discordjs/collection/pull/31/commits/b4765c3bf84ae00a73283293c1f3c306e44e45b6